pyproject.toml

```diff
@@ -0,0 +1,49 @@
+[build-system]
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "google_benchmark"
+description = "A library to benchmark code snippets."
+requires-python = ">=3.8"
+license = {file = "LICENSE"}
+keywords = ["benchmark"]
+
+authors = [
+    {name = "Google", email = "benchmark-py@google.com"},
```

Comment:
why isn't this the existing benchmark discussion group?

Reply:
This is meant to be more of a direct point of contact to the "authors" specifically, and it needs to be an email address.

See e.g. https://packaging.python.org/en/latest/specifications/declaring-project-metadata/#authors-maintainers

Reply:
benchmark-discuss@googlegroups.com :)

Reply:
Updated the email in 7fdfc4626e8de14130a0923e71b784f79ec6df93.